src/equations/compressible_euler_multicomponent_1d.jl

```diff
@@ -159,20 +160,21 @@ Source terms used for convergence tests in combination with
 @inline function source_terms_convergence_test(u, x, t,
                                                equations::CompressibleEulerMulticomponentEquations1D)
     # Same settings as in `initial_condition`
+    RealT = eltype(u)
     c = 2
-    A = 0.1
+    A = convert(RealT, 0.1)
     L = 2
-    f = 1 / L
-    omega = 2 * pi * f
+    f = 1.0f0 / L
+    omega = 2 * convert(RealT, pi) * f
 
     gamma = totalgamma(u, equations)
 
     x1, = x
     si, co = sincos((t - x1) * omega)
-    tmp = (-((4 * si * A - 4c) + 1) * (gamma - 1) * co * A * omega) / 2
+    tmp = (-((4 * si * A - 4 * c) + 1) * (gamma - 1) * co * A * omega) / 2
```

Comment:
I was wondering why `4c` was used here instead of `4 * c`. This is confusing as I took `4c` as a variable at the first place.

Reply:
👍